.travis.yml

```diff
@@ -22,6 +22,8 @@ before_install:
       fi
     - git config --global user.email gitsavvy@gitsavvy.com
     - git config --global user.name GitSavvy
+    - pip install --upgrade pip
+    - pip install pycodestyle
```

Comment:
you may want to move the code block https://github.com/divmain/GitSavvy/blob/99c54673fb617b86d869b628d6df064fae78ec23/.travis.yml#L37-L46
to here

Note that, on macOS, `pip` corresponds to python 2, so `pip3` is needed.

Reply:
```
pip install python-coveralls;
pip install pycodestyle;
```
vs
```
pip install python-coveralls pycodestyle;
```

Reply:
or
```
    - if [ "$TRAVIS_OS_NAME" == "osx" ]; then
            brew update;
            brew install python3;
      fi
    - pip3 install python-coveralls pycodestyle
```

Reply:
How did you check that? 
Am I not used to any CI?

Reply:
I just tried...